modules/transport-netty4/src/test/java/org/elasticsearch/http/netty4/Netty4HttpHeaderValidatorTests.java

```diff
@@ -138,7 +140,10 @@ public void testIgnoreReadWhenValidating() {
         assertNull("content should not pass yet, need explicit read", channel.readInbound());
 
         channel.read();
-        assertTrue(channel.readInbound() instanceof LastHttpContent);
+        var lastContent = channel.readInbound();
+        assertTrue(lastContent instanceof LastHttpContent);
+        ((LastHttpContent) lastContent).release();
+
```

Comment:
Or, in one line (with slightly nicer failure message too):

```suggestion
        asInstanceOf(LastHttpContent.class, channel.readInbound()).release();
```